test/with_dummyserver/async/test_poolmanager.py

```diff
@@ -1,14 +1,21 @@
 import io
 import json
 import pytest
+import time
 from ahip.base import DEFAULT_PORTS
 from ahip import PoolManager, Retry
-from ahip.exceptions import MaxRetryError, UnrewindableBodyError
+from ahip.exceptions import MaxRetryError, NewConnectionError, UnrewindableBodyError
+from hip.util.retry import RequestHistory
 
-from test.with_dummyserver import conftest
-from dummyserver.testcase import HTTPDummyServerTestCase
+from dummyserver.server import HAS_IPV6
+from dummyserver.testcase import HTTPDummyServerTestCase, IPv6HTTPDummyServerTestCase
 
 from test import LONG_TIMEOUT
+from test.with_dummyserver import conftest
+
+
+def _test_name(test_name, backend, anyio_backend):
```

Comment:
nit: does it make sense to use `"none"` since `anyio_backend` is a lowercase string?

Reply:
I had used "None" as it's `str(None)`, but lowercase is indeed more consistent. Done, thanks